code/modules/atmospherics/environmental/LINDA_turf_tile.dm

```diff
@@ -511,6 +529,18 @@
 		display_id = wrapping_id
 	thing.vis_contents += GLOB.colored_turfs[display_id]
 
+/datum/excited_group/proc/debug_total_moles()
+	var/moles = 0
+	for(var/turf/open/contained_turf as anything in turf_list)
+		var/delta_moles = 0
+		if(!contained_turf.air || !istype(contained_turf.air, /datum/gas_mixture/turf))
+			continue
+
+		TOTAL_MOLES(contained_turf.air.gases, delta_moles)
+		moles += delta_moles
+
+	message_admins("this excited group contained [length(turf_list)] turfs, the combined total of which contained [moles] moles of gas")
+
```

Comment:
??? (undocced change)

Reply:
oh right i was fucking around with excited groups and forgot to get rid of this